docs-translations/ko-KR/tutorial/planned-breaking-changes.md

```diff
@@ -0,0 +1,86 @@
+# API 변경 계획
+
+다음은 Electron 2.0 에서 제거될 API 목록입니다.
+
+정해진 적용 계획은 없지만 적어도 90일 전에 중단 경고를 할 것 입니다.
```

Comment:
Maybe just keep this top part ☝️ in Korean, then remove everything after it, replacing it with a link to the English list of deprecations:

```
[breaking changes](docs/tutorial/breaking-changes.md]
```